index.html

```diff
@@ -0,0 +1,184 @@
+                    <li class="nav-link" id="navlink">
+                        <a href="AboutPage.html" class="link">About</a>
+                    </li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">Join</a></li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">Sponsor</a></li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">News</a></li>
+                </ul>
+                <div class="baricon" id="nav-toggler">
+                    <div class="bar1"></div>
+                    <div class="bar2"></div>
+                    <div class="bar3"></div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="herosection">
+            <div class="herosection_content">
+                <div class="inner_container">
+                    <h4 class="text4">"Hello! Sharing World"</h4>
+                    <h2 class="text2">
+                        CREATIVITY WITH EXPERIENCE MATTERS IN LGB
+                    </h2>
+                </div>
+                <div class="notification_card">
+                    <span class="card_text">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quam aperiam quae similique, quisquam
+
+                    </span>
+                </div>
+
+                <div class="working_days_container">
+                    <h2 class="working_days">Monday ~ Sunday</h2>
+                    <span class="working_days_advert">@ a very affordable rate, with a clean a smart delivery</span>
+                </div>
+            </div>
+        </section>
+
+        <section class="main_programmer_section">
+            <div class="programmer_head">
+                <h3 class="service_header">
+                    Our Services
+                </h3>
+                <div class="decoration"></div>
+            </div>
+            <div class="card_container">
+                <div class="d-flex services_card">
+                    <img class="icon" src="/Assets/Icons/icon5.png" alt="icon">
+                    <h5 class="service">Decoration</h5>
+                    <div class="description">
+                        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex, tenetur!
+                    </div>
+                </div>
+                <div class="d-flex services_card">
+                    <img class="icon" src="/Assets/Icons/icon4.png" alt="icon">
+                    <h5 class="service">Cake Production</h5>
+                    <div class="description">
+                        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex, tenetur!
+                    </div>
+                </div>
+                <div class="d-flex services_card">
+                    <img class="icon" src="/Assets/Icons/icon3.png" alt="icon">
+                    <h5 class="service">Event Planning</h5>
```

Comment:
- [ ] Looking at the design of the `services` section. I think it needs more improvement in terms of the alignment. Please try to align the service cards in the center. Please look at the comparison 👇 

Your design
![image](https://user-images.githubusercontent.com/90222110/235301963-12ac427e-557a-46e1-88b3-18c6283cda4a.png)

Given design
![image](https://user-images.githubusercontent.com/90222110/235301989-7418c3c7-c1de-41ac-92c9-0d2bf6719f10.png)

- [ ] I want you to look at the design of the `see the whole program` button as we find on the design given. It is different from what you have in your design. However, it is important to note that this button has different styles depending on the mobile version and the desktop version. Please improve the design of that button on your website but make sure that button styles align with design given 👍